node/task.ts

```diff
@@ -911,7 +967,12 @@ export function find(findPath: string, options?: FindOptions): string[] {
 
                 if (options.followSymbolicLinks) {
                     // get the realpath
-                    let realPath: string = fs.realpathSync(item.path);
+                    let realPath: string;
+                    if (isUncPath(item.path)) {
+                        realPath = retry(fs.realpathSync, [item.path], { continueOnError: false, retryCount: 5 });
+                    } else {
+                        realPath = fs.realpathSync(item.path);
+                    }
```

Comment:
Could you please write some comments in the code regarding why we retrying if the provided path is UNC?